modules/lvms-creating-volume-clones.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-volume-clones_{context}"]
+= Creating volume clones
+
+To create a clone of a persistent volume claim (PVC), you must create a `PersistentVolumeClaim` custom resource (CR) in the namespace where you created the source PVC.
+
+[IMPORTANT]
+====
+The cloned PVC has write access.
+====
+
+.Prerequisites
+
+* You ensured that the source PVC is in `Bound` state. This is required for a consistent snapshot.
+
+.Procedure
+
+. Log in to the OpenShift CLI (`oc`).
+
+. Create a `PersistentVolumeClaim` custom resource (CR) YAML file.
++
+.Example `PersistentVolumeClaim` CR YAML file to create a volume clone
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: lvm-pvc-clone
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: lvms-vg1 <1>
+  volumeMode: Filesystem <2>
+  dataSource:
+    kind: PersistentVolumeClaim
+    name: lvm-pvc <3>
+  resources:
+    requests:
+      storage: 1Gi <4>
+----
+<1> Set this field to the value of the `storageClassName` field in the `PersistentVolumeClaim` CR of the source PVC.
+<2> To create a file PVC, set this field to `Filesystem`. To create a block PVC, set this field to `Block`.
+<3> Specify the name of the source PVC.
+<4> Specify the storage size for the cloned PVC. The storage size of the cloned PVC must be greater than or equal to the storage size of the source PVC.
+
+. Create the `PersistentVolumeClaim` CR in the namespace where you created the source PVC by running the following command:
++
+[source,terminal]
+----
+# oc create -f <file_name> -n <namespace>
+----
+
+.Verification
+
+* To verify that the volume clone is created, run the following command:
++
+[source,terminal]
+----
+$ oc get pvc -n <namespace>
+----
++
+.Example output
++
+[source, terminal]
+----
+NAME                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+lvm-block-1-clone   Bound    pvc-e90169a8-fd71-4eea-93b8-817155f60e47   1Gi        RWO            lvms-vg1       5s
+----
++
+The value of the `STATUS` field for the cloned PVC must be `Bound`.
```

Comment:
For the cloned PVC to get into the `Bound` state, it needs to be used by a Pod first.

Reply:
Removed this sentence. As it is not mandatory to be in Bound state. The cloned volume just needs to appear in the output of the `oc get pvc` command.